puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/builders/systest.yaml

```diff
@@ -0,0 +1,80 @@
+- builder:
+    name: systest
+    builders:
+      - shell: |+
+          #!/bin/bash -xe
+
+          # populate variables from builder macros if they're given
+          [ -n "{repo}" ] && repo="{repo}"
+          [ -n "{repo_environment}" ] && repo_environment="{repo_environment}"
+          [ -n "{pl_puppet}" ] && pl_puppet="{pl_puppet}"
+          [ -n "{run_hammer_tests}" ] && run_hammer_tests=$(echo "{run_hammer_tests}" | tr '[:upper:]' '[:lower:]')
+          [ -n "{db_type}" ] && db_type="{db_type}"
+          [ -n "{umask}" ] && umask="{umask}"
+          [ -n "{expected_version}" ] && expected_version=="{expected_version}"
```

Comment:
I think this should be `=` rather than `==`

Reply:
you are right!

Reply:
@ekohl fixed it!